Lesson31HomeWork/Lesson31HomeWork/ContentView.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  ContentView.swift
+//  Lesson31HomeWork
+//
+//  Created by Ernest Mihasenko on 26.06.22.
+//
+
+import SwiftUI
+
+struct ContentView: View {
+    var body: some View {
+        studentsInfoTextFields
+    }
+    
+    @State var studentList = [Student]()
+    
+    @State var name = ""
+    @State var surname = ""
+    @State var grade = ""
```

Comment:
It's better to declare your properties above `var body: some View`. Please, move them there